code/modules/reagents/chemistry/colors.dm

```diff
@@ -16,6 +16,8 @@ GLOBAL_LIST_INIT(random_color_list, list("#00aedb","#a200ff","#f47835","#d41243"
 	for(var/datum/reagent/reagent in reagent_list)
 		if(reagent.id == "blood" && reagent.data && reagent.data["blood_color"])
 			reagent_color = reagent.data["blood_color"]
+		if(reagent.id == "slimejelly" && reagent.data && reagent.data["colour"])
+			reagent_color = reagent.data["colour"]
```

Comment:
Why use this one instead of just `reagent.color`?

Reply:
Removed